source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
```

Comment:
Since we can’t link directly to Microsoft’s docs, what if we just said, “consult Azure’s documentation for more details about these values”?

That way we give at least _some_ answer, however abstract, to the question, “where do I find these values?”